containers/kubernetes/reference-content/multi-az-clusters.mdx

```diff
@@ -38,6 +38,17 @@ The main advantages of running a Kubernetes Kapsule cluster in multiple AZs are:
 
 For more information, refer to the [official Kubernetes best practices for running clusters in multiple zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/) documentation.
 
+## Limitations
+
+- Kapsule's control plane network access is managed by a Load Balancer in the primary zone of each region. If this zone fails globally, the Control Plane will be unreachable, even if the cluster spans multiple zones. This limitation also applies to HA Dedicated Control Planes.
+- Persistent volumes are limited to their Availability Zone (AZ). Applications must replicate data across persistent volumes in different AZs to maintain high availability in case of zone failures.
+- In "controlled isolation" mode, nodes access the Control Plane via their public IPs. If two AZs can't communicate (split-brain scenario), nodes won't appear unhealthy from Kubernetes' perspective, but communication between nodes in different AZs will be disrupted. Applications must handle this scenario if they use components across multiple AZs.
+- In "full isolation" mode, nodes also use the Public Gateway to access the Control Plane. If nodes can't reach the public gateway (such as private network failure in an AZ), they will become unhealthy. As there is only one Public Gateway per Private Network, losing the AZ with the Public Gateway results in the loss of all nodes in all private pools across all AZs.
```

Comment:
```suggestion
- In "full isolation" mode, nodes also use the Public Gateway to access the Control Plane. If nodes can't reach the Public Gateway (e.g. because of Private Network failure in an AZ), they will become unhealthy. As there is only one Public Gateway per Private Network, losing the AZ with the Public Gateway results in the loss of all nodes in all private pools across all AZs.
```